torch_xla/csrc/ops/dynamic_ir.cpp

```diff
@@ -196,6 +196,28 @@ int64_t SizeGe::getDynamicValue() const {
 
 std::string SizeGe::ToString() const { return "aten::size_ge"; }
 
+SizeGt::SizeGt(torch::lazy::Value a, torch::lazy::Value b)
```

Comment:
would be great to make sure `Lt` and `Le` are also ready to function without breaking us in the future. Please add as a separate PR.